### ISSUE TITLE
Project property page "Vary by configuration" gear menu icon is not keyboard accessible

### ISSUE THREAD

**Issue by japarson** [dotnet/project-system#7997]
## Visual Studio Version
Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview
Version 17.2.0 Preview 2.0

## Summary
The project property pages contain a gear icon which allows users to vary properties by certain conditions (e.g. target framework). This gear icon is not keyboard focusable, which is an accessibility issue.
- [Bug 1504137](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1504137): A11y_visual studio client_Code generation and run time section_SDK version_Keyboard: Menu button in "SDK version" section is not accessible with keyboard.

https://user-images.githubusercontent.com/59936622/159563384-c9842895-59cd-4709-8d3d-249064a9d31d.mp4

## Steps to Reproduce
1. Create new iOS project from power shell using `dotnet new ios -n <project_name>`.
1. Open VS 2022 and tab till "Open a project or solution" option and press enter.
1. Tab till "File name" and select newly created iOS "csproj" file and press enter on "Open" button.
1. Tab till project in "Solution explorer" and press context key.
1. Tab till "Properties" menu item and press enter.
1. Tab till "iOS Build" button and press enter.
1. Tab till "Build IPA" present in "IPA packaging options" section.
1. Tab till "SDK version" and verify whether Menu button is accessible with keyboard or not.

## Expected Behavior
The gear menu icon should be keyboard accessible.

## Actual Behavior
The gear menu icon is not keyboard accessible.

## User Impact
Keyboard users may feel inconvenient in accessing menu button if its not accessible with keyboard.

[MAS 2.1.1 – Keyboard](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b1F29ACF9-A3E7-414F-BD8D-7E3598D877BB%7d&file=MAS%202.1.1%20%E2%80%93%20Keyboard.docx&wdLOR=c6FA267E7-81CB-46D6-A357-F39B5ABB492F&action=default&mobileredirect=true)
